src/util.js

```diff
@@ -106,3 +108,59 @@ export function moveChildren(fromEl, toEl) {
     }
     return toEl;
 }
+
+export function transformCheckbox(to, from) {
+    if (to.checked || to.getAttribute('checked')) {
+        // force checked property
+        from.checked = true;
+    } else {
+        // force off checked property
+        from.checked = false;
+    }
+}
```

Comment:
The idea here would be to keep local elements property values in sync based on the server response.  So if you have locally a checkbox checked and the server (string only) says no checked attribute, update accordingly.